var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -101,7 +101,7 @@ def patch(self):
 
     @property
     def common_config_args(self):
-        return [
+        config_args = [
             '-prefix', self.prefix,
```

Comment:
Oh wow, so most of the common_config_args aren't compatible.

I don't know how I missed this, but it looks like qt3 does actually set the installation prefix already. In that case, this PR might be ready to go as is.